src/services/liking.service.js

```diff
@@ -13,6 +13,15 @@ const LikingService = {
     }
     like.update({ isLiked: !like.isLiked });
     return sendResult(res, 200, ` The accommodation like  status updated to ${like.isLiked}`, like);
+  },
+
+  async getLikes(req, res) {
+    const { accommodationId } = req;
+    const countLikes = await db.Likings
+      .findAndCountAll({ where: { accommodationId, isLiked: true }, raw: false });
+    const countUnlikes = await db.Likings
+      .findAndCountAll({ where: { accommodationId, isLiked: false }, raw: false });
+    return sendResult(res, 200, ` The ccommodation ${accommodationId} has:  ${countLikes.count} likes, ${countUnlikes.count} unlikes `);
```

Comment:
it will be useful on `frontend` if you format response `data` like this:
```
{
 like: countLikes,
 unlike: countUnlikes
}
```

Reply:
thank you @urbanishimwe, I will highly consider it